ProcessMaker/processmaker
# Implement Asterisk (*) for Required Fields in Project Select Lists

This PR adds the 'required' attribute to the ProjectSelect component when utilized within a Copy to Project model.

## Solution
- Included the 'required' attribute in the ProjectSelect component to ensure proper validation within the Copy to Project model.

## How to Test

1. Create a project within the application and add assets to it.
2. Access the project asset listing page and select 'Copy to Project' from the Ellipsis menu for a specific asset.
3. Confirm that the Project Select field is marked with the asterisk (*) symbol, indicating that it is a required field.

## Related Tickets & Packages
- [FOUR-11099](https://processmaker.atlassian.net/browse/FOUR-11099)
- [Projects Package PR](https://github.com/ProcessMaker/package-projects/pull/53)

ci:next
ci:package-projects:observation/FOUR-11099


## Code Review Checklist


[FOUR-11099]: https://processmaker.atlassian.net/browse/FOUR-11099?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
